e2e/shortcut-action/shortcut-action.e2e-spec.ts

```diff
@@ -58,19 +58,29 @@ describe('Shortcut-action App', function() {
   });
 
   it(`should move row up using 'mod+shift+up' shortcut`, () => {
-    let inputElem = page.getChildOfElementByCss(page.getElementById('accelerator_experiments.1.experiment'), 'textarea');
-    let inputElemValue = inputElem.getAttribute('value');
-    inputElem.sendKeys(protractor.Key.chord(mod, protractor.Key.SHIFT, protractor.Key.UP));
-    let movedUpInputElem = page.getChildOfElementByCss(page.getElementById('accelerator_experiments.0.experiment'), 'textarea');
-    expect(movedUpInputElem.getAttribute('value')).toEqual(inputElemValue);
+    let currentFirstRow = page.getElementArrayById('accelerator_experiments.0');
+    let currentSecondRow = page.getElementArrayById('accelerator_experiments.1');
+    let currentElem = page.getChildOfElementByCss(page.getElementById('accelerator_experiments.1.experiment'), 'textarea');
+    currentElem.sendKeys(protractor.Key.chord(mod, protractor.Key.SHIFT, protractor.Key.UP));
+    let targetFirstRow = page.getElementArrayById('accelerator_experiments.0');
+    let targetSecondRow = page.getElementArrayById('accelerator_experiments.1');
+    expect(targetFirstRow).toEqual(currentSecondRow);
+    expect(targetSecondRow).toEqual(currentFirstRow);
+    // Test for issue of overriding the new value with the old one when moving up because of triggering the commitValueChange()
+    // on blur with the old values. As a result the when moving up the switched cells has the same value
+    expect(targetSecondRow).not.toEqual(targetFirstRow);
   });
 
   it(`should move row down using 'mod+shift+down' shortcut`, () => {
-    let inputElem = page.getChildOfElementByCss(page.getElementById('accelerator_experiments.0.experiment'), 'textarea');
-    let inputElemValue = inputElem.getAttribute('value');
-    inputElem.sendKeys(protractor.Key.chord(mod, protractor.Key.SHIFT, protractor.Key.DOWN));
-    let movedDownInputElem = page.getChildOfElementByCss(page.getElementById('accelerator_experiments.1.experiment'), 'textarea');
-    expect(movedDownInputElem.getAttribute('value')).toEqual(inputElemValue);
+   let currentRow = page.getElementArrayById('accelerator_experiments.0');
```

Comment:
@jmartinm @harunurhan This is the helper function returning the row of elements by a specifc id. Any comments on this? I will update the PR to pass the build and just comment if we want something else to change.